components/scream/tests/physics/rrtmgp/rrtmgp_tests.cpp

```diff
@@ -1,7 +1,7 @@
 #define CATCH_CONFIG_MAIN
 
 #include "catch2/catch.hpp"
-#include "rrtmgp.hpp"
+#include <physics/rrtmgp/rrtmgp.hpp>
```

Comment:
Usually <> is for system includes, "" for project. This becomes signficant when using, e.g., the `-isystem` (or something like that) flag in GNU.

Reply:
Fair point. I'll find and change these occurrences.